docs/standard/microservices-architecture/architect-microservice-container-applications/asynchronous-message-based-communication.md

```diff
@@ -67,38 +67,38 @@ However, for mission-critical and production systems that need hyper-scalability
 
 A challenge when implementing an event-driven architecture across multiple microservices is how to atomically update state in the original microservice while resiliently publishing its related integration event into the event bus, somehow based on transactions. The following are a few ways to accomplish this, although there could be additional approaches as well.
 
--   Using a transactional (DTC-based) queue like MSMQ. (However, this is a legacy approach.)
+  - Using a transactional (DTC-based) queue like MSMQ. (However, this is a legacy approach.)
 
--   Using [transaction log mining](https://www.scoop.it/t/sql-server-transaction-log-mining).
+  - Using [transaction log mining](https://www.scoop.it/t/sql-server-transaction-log-mining).
 
--   Using full [Event Sourcing](https://msdn.microsoft.com/library/dn589792.aspx) pattern.
+  - Using full [Event Sourcing](https://msdn.microsoft.com/library/dn589792.aspx) pattern.
 
--   Using the [Outbox pattern](http://gistlabs.com/2014/05/the-outbox/): a transactional database table as a message queue that will be the base for an event-creator component that would create the event and publish it.
+  - Using the [Outbox pattern](http://gistlabs.com/2014/05/the-outbox/): a transactional database table as a message queue that will be the base for an event-creator component that would create the event and publish it.
 
-Additional topics to consider when using asynchronous communication are message idempotence and message deduplication. These topics are covered in the section [Implementing event-based communication between microservices (integration events)](#implementing_event_based_comms_microserv) later in this guide.
+Additional topics to consider when using asynchronous communication are message idempotence and message deduplication. These topics are covered in the section [Implementing event-based communication between microservices (integration events)](../multi-container-microservice-net-applications/integration-event-based-microservice-communications.md) later in this guide.
 
 ## Additional resources
 
--   **Event Driven Messaging**
-    [*http://soapatterns.org/design\_patterns/event\_driven\_messaging*](http://soapatterns.org/design_patterns/event_driven_messaging)
+-   **Event Driven Messaging**  
+    *http://soapatterns.org/design_patterns/event_driven_messaging*
```

Comment:
we cannot have bare URLs

Reply:
Oops, I "simplified" them all, to avoid possible inconsistencies 🙁

Reply:
Out of curiosity, do you know the reason for banning bare URLs if we need to display bare URLs?